vnext/Microsoft.ReactNative/Views/TextViewManager.cpp

```diff
@@ -25,10 +26,14 @@ namespace Microsoft::ReactNative {
 
 class TextShadowNode final : public ShadowNodeBase {
   using Super = ShadowNodeBase;
+  friend TextViewManager;
 
  private:
   ShadowNode *m_firstChildNode;
 
+  std::optional<winrt::Windows::UI::Color> m_ColorValue = std::nullopt;
```

Comment:
another option now that I think about it is to just use Colors::Transparent as signifying "not set" instead of optional. Not a huge deal but maybe preferrable :)
also, could you rename this to something that refers to the background? m_backgroundColor ?